README.md

```diff
@@ -1 +1,68 @@
-Library-JS
+# Library
+
+> In this project we build a small Library app to show and add books and set if you have already read it or not. 
+  ​
+- Every move is saved in the local storage of your browser.
+
+## Built With
+
+- HTML,
+- CSS
+- tailwind.css
+- Javascript,
+- VScode
+
+## Live Demo
+
+<a href="https://rawcdn.githack.com/SigmaSam/Library-JS/adf13634f6767c39fbe06ca410084dc48902ddbc/index.html" target="_blank">Live version</a>
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+- Clone this repository.
```

Comment:
- What's the command for cloning?
- After cloning then what? Add command for checking directory and checking out the `features` branch
- How do I open the app? Be explicit in your instructions. Don't assume the user knows how.